src/components/FriendList/FriendListHome.js

```diff
@@ -44,12 +42,17 @@ export default function FriendListHome(){
     }
 
     React.useEffect(()=> {
-        //user needs to be logged in to access
-        if (email === undefined || email === '') {
-            navigate("/login");
-        }
+        const email = JSON.parse(localStorage.getItem("email"));
+        axios.get(`${process.env.REACT_APP_BASE_URL}friend//incoming-requests-count/${email}`)
```

Comment:
Why does this request URL have // in `...friend//incoming...`?

Reply:
because it's a mistake